schema_cache.rst

```diff
@@ -0,0 +1,131 @@
+
++--------------------------------------------+-------------------------------------------------------------------------------+
+| Feature                                    | Required Metadata                                                             |
++============================================+===============================================================================+
+| :ref:`resource_embedding`                  | Foreign key constraints (to determine relationships between tables)           |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`Database Functions <s_procs>`        | Function's metadata (parameters, return type, volatility and                  |
+|                                            | `overloading <https://www.postgresql.org/docs/current/xfunc-overload.html>`_) |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`Upserts <upsert>`                    | Primary keys                                                                  |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`insert_update`                       | Primary keys (in order to return the Location header)                         |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`OPTIONS requests <options_requests>` | View primary key columns and INSTEAD OF TRIGGERS                              |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`open-api`                            | Table columns, primary keys and foreign keys                                  |
++                                            +-------------------------------------------------------------------------------+
+|                                            | View columns and INSTEAD OF TRIGGERS                                          |
++                                            +-------------------------------------------------------------------------------+
+|                                            | Function's metadata                                                           |
++--------------------------------------------+-------------------------------------------------------------------------------+
+
+The Stale Schema Cache
+----------------------
+
+When you make changes on the metadata mentioned above while PostgREST is running, the schema cache turns stale. Future requests you make related to these changes will need the :ref:`schema to be reloaded <schema_reloading>`; otherwise, you'll get an error instead of the expected result.
```

Comment:
```suggestion
When you make changes on the metadata mentioned above while PostgREST is running, the schema cache turns stale. Future requests that use the above features will need the :ref:`schema to be reloaded <schema_reloading>`; otherwise, you'll get an error instead of the expected result.
```